network/p2pPeer.go

```diff
@@ -65,8 +65,9 @@ func (c *wsPeerConnP2PImpl) WriteMessage(_ int, buf []byte) error {
 	return err
 }
 
+// Do nothing for now since this doesn't actually close the connection just sends the close message
 func (c *wsPeerConnP2PImpl) CloseWithMessage([]byte, time.Time) error {
-	return c.stream.Close()
```

Comment:
Do we want to actually send a websocket close message here that we can read from wsPeer.readLoop()?